resources/definitions/ultimaker_method_base.def.json

```
@@ -395,25 +396,30 @@
         "speed_wall_0": { "value": "speed_wall * 30/40" },
         "speed_wall_x": { "value": "speed_wall" },
         "support_angle": { "value": 40 },
-        "support_bottom_distance": { "value": "support_z_distance / 2" },
+        "support_bottom_distance": { "value": "layer_height if extruder_nr == support_extruder_nr else 0" },
+        "support_bottom_enable": { "value": "false if extruder_nr == support_extruder_nr else true" },
         "support_bottom_material_flow": { "value": "material_flow" },
+        "support_bottom_wall_count": { "value": "0 if extruder_nr == support_extruder_nr else support_wall_count" },
         "support_brim_enable": { "value": false },
         "support_conical_min_width": { "value": 10 },
         "support_enable": { "value": true },
         "support_extruder_nr": { "value": "int(anyExtruderWithMaterial('material_is_support_material'))" },
-        "support_fan_enable": { "value": false },
+        "support_fan_enable": { "value": "true if extruder_nr == support_extruder_nr else false" },
```

This can be written in a simpler way:
```suggestion
        "support_fan_enable": { "value": "extruder_nr == support_extruder_nr" },
```